### PR TITLE
260 Render supervisors approvals page correctly when no approval data is available.

### LINKED ISSUE #260: Supervisors approvals page returns a 500 error page when no data is available.

### PR DESCRIPTION
**Pull request by JohnMwashuma**
Fixes #260

### CODE REVIEW

File: tally_ho/apps/tally/views/reports/staff_performance_metrics.py [hunk 1]
```diff
@@ -53,10 +53,13 @@ def approvals_percantage(self, approvals):
         """
         forms_approved = approvals['forms_approved']
         forms_sent_for_review = approvals['forms_sent_for_review']
+        decimal_digits = 2
```
Comment (moshthepitt) [inline tally_ho/apps/tally/views/reports/staff_performance_metrics.py]: Should we make this a param to this function?

File: tally_ho/apps/tally/templates/reports/supervisor_approvals.html [hunk 2]
```diff
@@ -13,7 +13,7 @@ <h3>{% trans "Audit Supervisors Approval Rate" %}</h3>
         <th>{% trans 'Number of Forms Sent For Review' %}</th>
         <th>{% trans 'Approval Rate' %}</th>
     </tr>
-    {% if a_s_approvals.forms_approved == 0 %}
+    {% if not a_s_approvals.forms_approved %}
     <tr>
         <td>No Data</td>
```
Comment (moshthepitt) [inline tally_ho/apps/tally/templates/reports/supervisor_approvals.html]: Do we want to make this string translatable?

File: tally_ho/apps/tally/views/reports/staff_performance_metrics.py (hunk 1 above)
Comment (JohnMwashuma) [reply tally_ho/apps/tally/views/reports/staff_performance_metrics.py]: makes sense, will do

File: tally_ho/apps/tally/templates/reports/supervisor_approvals.html (hunk 2 above)
Comment (JohnMwashuma) [reply tally_ho/apps/tally/templates/reports/supervisor_approvals.html]: We do

Comment (JohnMwashuma) [reply tally_ho/apps/tally/templates/reports/supervisor_approvals.html]: Done

File: tally_ho/apps/tally/views/reports/staff_performance_metrics.py (hunk 1 above)
Comment (JohnMwashuma) [reply tally_ho/apps/tally/views/reports/staff_performance_metrics.py]: Done

**Pull request merged**